### PR TITLE
tidb: accelerate unit test

### PR DESCRIPTION
**Pull request by tiancaiamao**
Avoid BootstrapSession everytime, just do it once in testSessionSuite.SetUpSuite

Before this commit,  unit test for tidb package cost 202s on my machine, now it takes 47s

### CODE REVIEW

**Comment by tiancaiamao** [comment]
It also reduce the "db is closed" error message.

File: ddl/ddl_db_test.go [hunk 1]
```diff
@@ -82,6 +84,7 @@ func (s *testDBSuite) TearDownSuite(c *C) {
 	localstore.MockRemoteStore = false
 	s.s.Execute("drop database if exists test_db")
 	s.s.Close()
+	s.dom.Close()
```
Comment (coocood) [inline ddl/ddl_db_test.go]: Close domain before store?

Comment (tiancaiamao) [inline ddl/ddl_db_test.go]: DONE

**Comment by coocood** [comment]
LGTM

**Comment by tiancaiamao** [comment]
PTAL @zimulala @hanfei1991 @shenli

File: tidb_test.go [hunk 2]
```diff
@@ -64,19 +63,21 @@ func (s *brokenStore) Open(schema string) (kv.Storage, error) {
...
+	s.store = newStore(c, s.dbName)
+	dom, err := BootstrapSession(s.store)
+	c.Assert(err, IsNil)
+	s.dom = dom
 }
 
 func (s *testMainSuite) TearDownSuite(c *C) {
 	defer testleak.AfterTest(c)()
+	s.dom.Close()
+	s.store.Close()
```
Comment (zimulala) [inline tidb_test.go]: Add error checking.

**Comment by zimulala** [comment]
LGTM

**Pull request merged**